public/data/Kendall_Does_Coding.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Abdus Salam",
```

Comment:
I think the name here should be Kendall Pinto 😅 am I right?
```suggestion
  "name": "Kendal Pinto",
```

Reply:
Lol, I just took it from his template and did the needed edits LOL

Reply:
Thanks